content/CORE/CORETutorials/Storage/Pools/StorageEncryption.md

```diff
@@ -251,7 +251,7 @@ This method does not preserve file ACLs.
 {{< /hint >}}
 The web interface supports using **Tasks > Rsync Tasks** to transfer files out of the GELI pool.
 {{< expand "File Transfer Method" "v" >}}
-In the **Shell**, `rsync` and other file transfer mechanisms (`scp`, `cp`, `sftp`, `ftp`, `rdiff-backup`) are available for copying data between pools.
+In the shell, `rsync` and other file transfer mechanisms (`scp`, `cp`, `sftp`, `ftp`, `rdiff-backup`) are available for copying data between pools.
```

Comment:
It's not a CORE shell really though, it's whatever shell they are using (BASH, zsh, etc). I'm removing the emphasis because they referred to the **Shell** screen, which will no longer exist in 13.1 (except for within a jail, at least according to Stavros).

Reply:
perfect!